config/config.go

```diff
@@ -132,13 +132,23 @@ func GetOptionalParameters(config *Config, defaultLidarDataRateMsec, defaultIMUD
 			optionalConfigParams.ImuName = imuName
 			strMovementSensorDataFreqHz, ok := config.MovementSensor["data_frequency_hz"]
 			if !ok {
```

Comment:
mind changing `ok` to `exists` while you're here?

Reply:
will do